--- re2-2020-08-01/Makefile.orig	2020-09-26 09:45:26.973445896 +0200
+++ re2-2020-08-01/Makefile	2020-09-26 09:49:34.678770628 +0200
@@ -260,8 +260,11 @@
 .PHONY: testofiles
 testofiles: $(TESTOFILES)
 
+.PHONY: compile-test
+compile-test: $(DTESTS) $(TESTS) $(STESTS)
+
 .PHONY: test
-test: $(DTESTS) $(TESTS) $(STESTS) debug-test static-test shared-test
+test: compile-test debug-test static-test shared-test
 
 .PHONY: debug-test
 debug-test: $(DTESTS)
